### ISSUE TITLE
Postman console lags behind Collection Runner on Desktop Application

### ISSUE THREAD

**Issue by nrmartell** [a85/POSTMan-Chrome-Extension#9929]
**Describe the Issue**
After a certain amount of requests run the Postman Console begins to lag behind Collection Runner. This more noticeable on Bigger collections with a lot of requests. At around the 70th request or so the Postman Console and Collection Runner begin to get out of sync. 

After the run stops, the logs are still updating the console cannot be cleared to see the latest request you must wait until run is complete. Depending on the number of requests in the collection, this can be time consuming. 

**Steps to Reproduce**
Run a collection with 100+ requests

**Screenshots or Videos**
https://www.loom.com/share/15c192bc888c4e31b4347d9b966d1184 

**Environment Information**

 - Platform Type: Desktop
 - Postman Version: 8.3.1
 - Operating System: Windows/OSX

**Comment by ManiacDC** [comment]
Yes, this is the reason behind this issue: https://github.com/postmanlabs/postman-app-support/issues/9476

Very irritating as I have thousands of requests in my collection.
